docs/usage.rst

```diff
@@ -4,9 +4,135 @@ Usage
 Creating Virtual Machines
 -------------------------
 
-Create vms
+Follow the steps carefully so that you dont get any error while creating virtual machine. The steps are very easy only need is to follow them in order.
```

Comment:
so that you `don't`, and don't write creating use the word `provisioning`.
Instead of `The steps are very easy only need is to follow them in order` just write `Follow the below steps`